src/index.html

```diff
@@ -17,16 +17,27 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="searchbar"
+      data-qa="big"
+    >
+      <input
+        class="searchbar__input searchbar__input-big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
+    <form
+      data-qa="small"
+      class="searchbar"
+    >
```

Comment:
Similar to the previous form, this form tag should not be self-closing. Use `<form>...</form>` instead.